packages/@react-spectrum-uxp/storybook/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "rs3uxp-plugin",
+  "version": "3.0.0-alpha.1",
+  "private": true,
+  "license": "MIT",
+  "main": "dist/index.html",
+  "scripts": {
+    "watch:parcel": "rm -rf ../../../.parcel-cache && parcel serve src/index.html --target main --no-content-hash --no-cache --log-level verbose",
```

Comment:
so to run uxp storybook, you'd expect a user to be at the root and use `lerna run --scope rs3uxp-plugin watch:parcel`? or do you expect them to be in this directory? or how does this work?

Reply:
you don't need to rm -rf the parcel-cache, just running parcel with the flag `--no-cache` that you have already should be fine
did you run into an issue with that?

Reply:
I've run into a million issues with the parcel cache, so I always delete it as a standard parcel ritual. I guess --no-cache will do the trick though.

Concerning a user running storybook, currently, I run it from that directory. To start with, only the RS3-UXP developers will be running the storybook. As we get farther along, we will improve the documentation, improve the storybook UI and eventually we will provide instructions for external developers to be able to run the storybook as a plugin within XD (or other UXP plugin compatible product).

I know there is a ton of work to do before we reach that point. Initially I'm just trying to get the basic structure right so that I can open up RS3+UXP development to the main team that will be implementing the components.